simcon_project/conversation_templates/urls/create_assignment_urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from conversation_templates.views import *
+
+app_name = 'assignments'
+urlpatterns = [
+    path('', CreateAssignmentView.as_view(), name="create_assignment"),
+    path('add_data/', add_data, name="add_data"),
```

Comment:
Make sure to use the naming convention we went over on the weekend. Names should be "create-assignment" and "add-data" with - instead of _